src/main/java/org/hmhb/program/Program.java

```diff
@@ -0,0 +1,159 @@
+package org.hmhb.program;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.hmhb.organization.Organization;
+
+@Entity
+public class Program {
```

Comment:
So is the plan that later in the next version we will add a RequestedVersion, or perhaps just add a published flag to this object?